src/Api/RequestAttributesExtractor.php

```diff
@@ -29,19 +31,17 @@
      *
      * @throws RuntimeException
      *
-     * @return array
+     * @return array [$resourceClass, $collectionOperation, $itemOperation, $format]
```

Comment:
I just pushed a new commit with this change an a refactoring of request attributes usage.